src/main/java/com/github/calebwhiting/runelite/plugins/actionprogress/detect/ChatboxDetector.java

```diff
@@ -278,8 +278,7 @@ public void setup()
 		 */
 		this.registerAction(FLETCH_ATTACH, Fletching.UNENCHANTED_BOLTS_AND_ARROWS);
 		this.registerAction(FLETCH_ATTACH, HEADLESS_ARROW, FLIGHTED_OGRE_ARROW, AMETHYST_BROAD_BOLTS, AMETHYST_ARROW);
-		this.registerAction(FLETCH_JAVELIN_1Tick, Fletching.JAVELINS);
-		this.registerAction(FLETCH_JAVELIN_2Tick, IRON_JAVELIN, MITHRIL_JAVELIN);
+		this.registerAction(FLETCH_JAVELIN_2Tick, Fletching.JAVELINS);
```

Comment:
The 2 tick could be removed from the name since there is no longer a need to differentiate, but it's not a big deal